pkg/repro/repro.go

```diff
@@ -846,6 +847,13 @@ var cSimplifies = append(progSimplifies, []Simplify{
 		opts.EnableBinfmtMisc = false
 		return true
 	},
+	func(opts *csource.Options) bool {
+		if !opts.EnableCloseFds || opts.Repeat {
```

Comment:
Please add a comment here explaining the reason behind the opts.Repeat part. It's super WAT.

Reply:
Done.